docs/v1/milestones.md

```diff
@@ -0,0 +1,55 @@
+# v1 Milestones
+
+**NOTE: THIS DOC IS A WORK IN PROGRESS**
+
+This document captures concensus on the identity and content of milestones.
+
+## Legos-1
+
+Tentative target date: Feb 1, 2017
+
+1.  `servicecatalog/v1alpha1` API complete and functional:
+  1.  `Broker`, `ServiceClass`, `Instance`, `Binding`
+  2.  Integrators should be able to program against this REST API
+2.  API server serves `servicecatalog/v1alpha1`
+3.  Golang client for API in (1) is auto-generated
+
+## MVP-1
+
+Tentative target date: March 1, 2017
+
+The MVP1 milestone is the barest skeleton of an MVP that has the right
+architecture.
+
+High-level functional requirements:
+
+1.  `servicecatalog/v1alpha1` API complete and functional:
+  1.  `Broker`, `ServiceClass`, `Instance`, `Binding`
+  2.  Integrators should be able to program against this REST API
+2.  Golang client for API in (1) is auto-generated
+3.  Native-k8s-style CLI experience
+  1.  Formatted list
+  2.  Formatted describe
+4.  Integration with Open Service Broker API:
+  1.  Add broker
+  2.  Provision service instance
+  3.  Bind to service instance
```

Comment:
I can't recall where we agreed they would go, but a single secret is fine with me.